static_src/util/cf_api.js

```diff
@@ -386,9 +386,24 @@ export default {
       .then(() => {
         userActions.deletedUser(userGuid, orgGuid);
       }).catch((err) => {
-        if (err.response.data) {
-          userActions.errorRemoveUser(userGuid, err.data);
+        // Check whether we got caught on user roles in spaces
```

Comment:
I think we can probably move this error handling up into the calling function, it gives us a little more flexibility in case we want to handle the error differently in different contexts.